server/src/completions/snippets.ts

```diff
@@ -0,0 +1,130 @@
+/**
+ * Inspired by bash-language-sever
+ * Repo: https://github.com/bash-lsp/bash-language-server
+ */
+
+import { InsertTextFormat, type CompletionItem, CompletionItemKind, MarkupKind } from 'vscode-languageserver'
+
+export const SNIPPETS: CompletionItem[] = [
+  {
+    documentation: 'Fetch something',
```

Comment:
something is a bit vague. You should copy the description from the bitbake manual in all documentations here: https://docs.yoctoproject.org/4.2.3/singleindex.html#do-fetch

Reply:
yes, the `documentation` here is just dummy content, I think it is better to remove them as I am planning to extract them from the docs (like Io did with `onHover`) when the completion is resolved (`connection.onCompletionResolved()`). It is already in my todo list.